pages/language-tour.md

```diff
@@ -278,35 +231,31 @@ public fun main() {
 
 The type of a binding is determined at declaration.
 If an initializing expression is present, such as in all previous examples, the binding is given the type of that expression.
-Alternatively, we may state the type of a binding explicitly by the means of an annotation:
+Alternatively, we may state the type of a binding explicitly:
 
 ```val
 public fun main() {
-  var weight: Double = 1
+  var weight: Double = 1.0
   weight = 2.3
   print(weight) // 2.3
 }
 ```
+
+The type of an expression can be retrieved, without evaluating the expression, using `type(of:)`:
 {% comment %}
-The ability to use (e.g.) an integer literal to initialize a double is IIUC one of the great sources of expression too complex errors in Swift.
+I understand, precedent from Swift, but… what purpose does `of:` serve here?!
```

Comment:
I was about to change and then it felt wrong when I saw `type(weight)` in the code. I guess I'm used to `typeof` javascript, so the label seems natural to me.

Reply:
Yeah, other languages too.